source/connection-strings/relational-database-connection-strings.txt

```diff
@@ -1,3 +1,194 @@
+     - ``prodHost``
+   * - Port
+     - ``1521``
+   * - SID (System Identifier)
+     - ``ORCL``
+
+.. note::
+
+   To learn more about Oracle connection strings, see:
+
+   - `Oracle JDBC Developer's Guide and Reference <https://docs.oracle.com/cd/B28359_01/java.111/b31224/jdbcthin.htm>`__
+   - `Connection String Attributes <https://docs.oracle.com/cd/E85694_01/ODPNT/ConnectionConnectionString.htm#GUID-DF4ED9A3-1AAF-445D-AEEF-016E6CD5A0C0__BABBAGJJ>`__
+
+PostgreSQL
+----------
+
+The general form for a PostgreSQL connection string is:
+
+.. code-block::
+
+   jdbc:postgresql://<host:port>/<database>?<properties>
+
+For example, consider the following connection string:
+
+.. code-block::
+
+   jdbc:postgresql://localhost:5432/pg-demo
+
+The preceding connection string specifies these connection details:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Property
+     - Value
+   * - Host
+     - ``localhost``
+   * - Port
+     - ``5432``
+   * - Database
+     - ``pg-demo`` 
+
+.. note::
+
+   To learn more about PostgreSQL connection strings, see:
+
+   - `PostgreSQL Connection Configuration Properties <https://jdbc.postgresql.org/documentation/use/>`__
+
+SQL Server
+----------
+
+The general form for a SQL Server connection string is:
+
+.. code-block::
+
+   jdbc:sqlserver://[serverName[\instanceName][:portNumber]][;property=value[;property=value]]
+
+For example, consider the following connection string:
+
+.. code-block::
+
+   jdbc:sqlserver://localhost:1433;databaseName=test
+
+The preceding connection string specifies these connection details:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Property
+     - Value
+   * - Host
+     - ``localhost``
+   * - Port
+     - ``1433``
+   * - databaseName
+     - ``test``
+
+**Using Windows Integrated Authentication**
+
+To enable Windows Integrated Authentication, add ``integratedSecurity=true;`` to the URI options.
+Leave the :guilabel:`Username` and :guilabel:`Password` fields blank. Windows Integrated Authentication connects 
+to the database using the credentials of the user who launched the Relational Migrator executable.
+
+**Using TLS**
+
+JDBC connections to SQL Server use Transport Layer Security (TLS) by default.
+The encrypt property controls TLS. To disable it, set ``encrypt=false;``.
+When TLS is enabled, the driver tries to validate the server's certificate by default.
+To implicitly trust the server certificate, set ``trustServerCertificate=true;``.
+
+.. note::
+
+   To learn more about SQL Server connection strings, see:
+
+   - `Setting Connection Properties <https://learn.microsoft.com/en-us/sql/connect/jdbc/setting-the-connection-properties>`__
+   - `SQL Docs: Building the Connection URL <https://docs.microsoft.com/en-us/sql/connect/jdbc/building-the-connection-url?view=sql-server-ver15>`__.
+   - `Connecting to SQL Server with the JDBC Driver <https://docs.microsoft.com/en-us/sql/connect/jdbc/connecting-to-sql-server-with-the-jdbc-driver?view=sql-server-ver15>`__
+
```

Comment:
(titles)
It's ok to omit font treatment from titles.

Reply:
@ianf-mongodb, yes, absolutely maintain the casing for properties, commands, etc. Thank you!